Stanford/JumpstartLab/Install/PathologicSettings.php

```diff
@@ -21,6 +21,7 @@ public function execute(&$args = array()) {
     $paths = array(
       'https://sites.stanford.edu/jsa-content/',
       '/jsa-content',
+      '/hslabs',
```

Comment:
I changed the path to /jsa-content instead of /hslabs as that site may get retired in the near future. You can remove the pathologic option here now.